### DISCUSSION TITLE
READER RETUNR NOT FOUND

### DISCUSSION THREAD

**Discussion by Mtehabsim** [cogeotiff/rio-tiler#639] [Q&A]
I pass local file to the reader it return file not found but I am sure of the path
also i used the same code with the same file two weeks ago and it worked just fine, did you guys change anything?

**Comment by vincentsarago** [comment]
Dear @Mtehabsim,

First there is no need to use capital letter for the whole title! Second without proper reproductible code and more info about rio-tiler version or system info we can't help you 🙏 

Thank you
